### PR TITLE
Review updates

### PR DESCRIPTION
**Pull request by jordaninthewind**

### CODE REVIEW

File: src/task1-1.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-  return str.split("").reverse().join("")
+  return str
+    .split('')
+    .reverse()
+    .join('');
 }
 
 function askQuestion() {
-  rl.question("Type input: \n", input => {
+  rl.question('Type input: \n', (input) => {
```
Comment (MurDaD) [inline src/task1-1.js]: In output you;'re getting these lines in 1 iteration:
```
<input>
<input>
<output>
```
Try to think how you can remove the middle `<input>`. Note: good idea to use `process.stdin.on`

File: src/task1-1.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
   input: process.stdin,
-  output: process.stdout
+  output: process.stdout,
 });
 
-askQuestion();
-
 function reverse(str) {
-  return str.split("").reverse().join("")
+  return str
```
Comment (MurDaD) [inline src/task1-1.js]: this function returns immediately - you can convert it to arrow function

**Comment by MurDaD** [comment]
Package name should contain no capital letters, so "module-1" should it be

**Comment by MurDaD** [comment]
1.2 crashes because of error:
```
Error: ENOENT: no such file or directory, open 'output/formattedCsv.txt'
Emitted 'error' event on WriteStream instance at:
    at internal/fs/streams.js:294:12
    at FSReqCallback.oncomplete (fs.js:146:23) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'output/formattedCsv.txt'
}```

YOu need to either create `./output` folder, or keep it in repo with `.gitkeep` file

File: package.json [hunk 3]
```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "task:1.1": "nodemon ./task1-1.js",
-    "task:1.2": "nodemon ./task1-2.js",
-    "task:1.3": "nodemon ./task1-3.js"
+    "lint": "eslint **.js",
+    "task:1.1": "nodemon ./src/task1-1.js",
+    "task:1.2": "nodemon ./src/task1-2.js",
+    "task:1.3": "nodemon ./lib/index.js"
```
Comment (MurDaD) [inline package.json]: `"nodemon ./src/task1-3.js",`

File: src/task1-3.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import csvtojson from 'csvtojson';
```
Comment (MurDaD) [inline src/task1-3.js:1]: I believe, we should not use TypeScript in first module, but anyway, script crashes with `SyntaxError: Cannot use import statement outside a module` exception

File: src/task1-1.js (hunk 1 above)
Comment (jordaninthewind) [reply src/task1-1.js]: I noticed this and couldn't work out why it was happening. The readline package was a little mysterious, so I took your note and updated to process.stdin.on... I didn't realize that 'data' would submit itself entirely as a chunk from the CLI.

File: src/task1-1.js (hunk 2 above)
Comment (jordaninthewind) [reply src/task1-1.js]: Noted and updated. Linter split it into lines, but I'll always prefer arrow functions when then fit.

File: package.json (hunk 3 above)
Comment (jordaninthewind) [reply package.json]: So, that was meant to be the compiled run after the babel command listed in the README. I've updated accordingly and brought it the one command.

File: src/task1-3.js (hunk 4 above)
Comment (jordaninthewind) [reply src/task1-3.js:1]: This was meant to be the es6 modules syntax and not TypeScript, I'd love to use TypeScript in this, but I'm woefully unaware of it. I have the time to try, though, fwiw.

**Pull request closed without merging**